Packs/CTIX/Integrations/CTIXv3/CTIXv3.py

```diff
@@ -1405,27 +1478,69 @@ def get_conversion_feed_source_command(
 def get_lookup_threat_data_command(
     client: Client, args: Dict[str, Any]
 ) -> List[CommandResults]:
-    '''
+    """
     Get Lookup Threat Data Command
 
     :Description Get Lookup Threat Data
     :param Dict[str, str] args: Paramters to be send to in request
     :return CommandResults: XSOAR based result
-    '''
+    """
     object_type = args.get("object_type", "indicator")
+    ioc_type = args.get("ioc_type", [])
```

Comment:
Since this is a list, I would recommend using the CommonServerPython function- argToList()

Reply:
Fixed. Thanks!